### DISCUSSION TITLE
Curve in point cloud

### DISCUSSION THREAD

**Discussion by DeDeckerThomas** [ifm/ifm3d#376] [Q&A]
Hello everyone! :wave: 

I have an O3R225 camera and I've noticed that the output point cloud for a given flat plane surface "curves" a bit. It looks like noise at first, but after some research I feel like it has something to do with the camera distortion. I can't really find anything about the lens distortion of this particular camera in the O3R documentation. Is there a way to disable or correct this?

Thanks in advance!
Thomas

**Comment by lola-masson** [comment]
Hi @DeDeckerThomas,
We are in the process of updating all the optical models for the cameras to use the fish eye model for O3R225 instead of the current one (Bouguet model). This could be the reason for a slight distortion in the cloud. However, you really should not see something too significant even with the "old" optical model. Are you able to share data or a screenshot of the flat surface in the point cloud, just so that we can double check you get what we expect.

**Reply by DeDeckerThomas** [reply]
Hi @lola-masson .
I can share some data by email. What is your email?

**Reply by lola-masson** [reply]
Great, you can send data to support.robotics@ifm.com.
